frontend/src/components/MainMenu/MainMenu.jsx

```diff
@@ -53,6 +53,7 @@ class MainMenu extends React.Component {
               key={index}
               page={menu.page}
               active={menu.expanded}
+              collapsed={this.props.isMainMenuExpanded}
```

Comment:
I changed this to isExpanded and used (NOT isExpanded) as it was working oppositely without the '!' --> if something is not expanded, it's collapsed.